wallet/chain/x/builder_test.go

```diff
@@ -0,0 +1,583 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package x
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/slices"
```

Comment:
We should be able to use the stdlib slices pkg here

Reply:
Thanks for spotting this. Fixed!